### DISCUSSION TITLE
Support for STR-DN860 Receiver?

### DISCUSSION THREAD

**Discussion by iceddonut** [kovalev-sergey/homebridge-sony-audio#12] [General]
Hi. Any plans to support the STR-DN860? From what I can tell, it's basically has the same functions as STR-DN1080, same API...etc but no HDR support. If there anyway i can get it working?

**Comment by kovalev-sergey** [comment]
Hi,
Only these https://developer.sony.com/develop/audio-control-api/documentation/supported-devices/  devices are supported.

STR-DN860 does not support audio control api. 


Sergey

On 7 Dec 2021, at 04:21, Iceddonut ***@***.***> wrote:

﻿
Hi. Any plans to support the STR-DN860? From what I can tell, it's basically has the same functions as STR-DN1080, same API...etc but no HDR support. If there anyway i can get it working?

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
